### ISSUE TITLE
module has no exported member

### ISSUE THREAD

**Issue by polaroidkidd** [mrmckeb/typescript-plugin-css-modules#87]
**Praise for the Project**
I've been looking around for a while seeing if there's any solid solutions to having scss modules typed and came across your project. It's fantastic! 

**Describe the bug**
I'm not sure if I configured the plugin correctly, if I'm missusing it or if I'm expecting something from it which it doesn't solve. Basically, I'm using nested scss classes and want to use named imports, but TS claims that the scss module does not have an exported. Here's the specific [error](https://github.com/polaroidkidd/dle.dev/blob/15fd71d7e58f776d31560a1420565dff5986f99c/src/app/components/profile-picture/profile-picture.tsx#L2-2), the [ts-config](https://github.com/polaroidkidd/dle.dev/blob/36ff59c1df13aef4a6b4259c6b1e8b0d0de5b43a/tsconfig.json#L12), the [scss file](https://github.com/polaroidkidd/dle.dev/blob/36ff59c1df13aef4a6b4259c6b1e8b0d0de5b43a/src/app/components/profile-picture/profile-picture.module.scss#L1). It also logs the imported class as `undefined` when [i log it here](https://github.com/polaroidkidd/dle.dev/blob/15fd71d7e58f776d31560a1420565dff5986f99c/src/app/components/profile-picture/profile-picture.tsx#L26).

**To Reproduce**

1. Clone https://github.com/polaroidkidd/dle.dev/tree/feat/migrate-to-custom-webpack
2. Check out `feat/migrate-to-custom-webpack`
3. `yarn install && yarn start`

**Expected behavior**
I am under the impression that the camelCaseOnly flag generates types, not only from classes but also from nested classes which use the BEM notation.

**Comment by mrmckeb** [comment]
Hi @polaroidkidd, thanks for the feedback!

I see that you bootstrapped this project with `create-react-app` and ejected. Can I ask why? I'm interested because I'm also on the Create React App team and we always encourage people to reject only as a last resort. Without ejecting, this plugin will work fine with CRA.

Due to limitations in TypeScript, this plugin is designed to give you errors in your IDE, but can't give type errors during build. The TypeScript team haven't committed to a timeframe for this, or even that they'll support it sadly.

Looking at your project, there doesn't seem to be any Webpack configuration to handle Sass/CSS at all... and this is why you're getting that error. 

As you started with CRA, you could reference our config to see how we implement it:
https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js

There are also many guides available (this was the first result on Google):
https://developerhandbook.com/webpack/how-to-configure-scss-modules-for-webpack/

Let me know if I can offer any more help!

**Comment by polaroidkidd** [comment]
I've been meaning to get back to you with this, but life got in the way. Sorry for the wait.

As to why I ejected, I wanted to learn more about webpack and bundling so I took it as a learning opportunity.  The configurations for the css/scss are in the `build-utils/webpack/development.webpack.ts` files.


That's a real shame. 
> Due to limitations in TypeScript, this plugin is designed to give you errors in your IDE, but can't give type errors during build. The TypeScript team haven't committed to a timeframe for this, or even that they'll support it sadly.

Thanks for looking into it though!
